### PR TITLE
ns scale fix

### PR DESCRIPTION
**Pull request by PrasadhNanjundan05**

### CODE REVIEW

File: mano-core-api/src/main/java/com/ubiqube/etsi/mano/service/NsScaleStrategyV3.java [hunk 1]
```diff
@@ -189,7 +189,7 @@ private static int getBaseStep(final NsdInstance instance, final String aspectId
 			return 0;
 		}
 		return nsScale.stream()
-				.filter(x -> x.getAspectId().equals(aspectId))
+				.filter(x -> (x.getAspectId() != null && x.getAspectId().equals(aspectId)))
```
Comment (Vignaudo) [inline mano-core-api/src/main/java/com/ubiqube/etsi/mano/service/NsScaleStrategyV3.java:192]: I don't understand this one.
In SOL003 scaling, `asspectId` is mandatory.
Same in SOL001, for the definition of scale levels, and so on ...

File: nfvo-core/src/main/java/com/ubiqube/etsi/mano/nfvo/notification/vnfind/VnfIndicatorValueChangeNotificationImpl.java [hunk 2]
```diff
@@ -385,13 +385,13 @@ public void callAction(final String vnfInstanceId, final String nsInstanceId, fi
 			ScaleNsData snd = new ScaleNsData();
 			snd.setScaleNsByStepsData(snbsd);
 			nsInst.setScaleNsData(snd);
-			LOG.info("NS Scale {} : {} launched", nsInstanceId, nsInst);
+			LOG.info("NS Scale {} {} : {} launched", snbsd.getScalingDirection().toString(), nsInstanceId, nsInst);
```
Comment (Vignaudo) [inline nfvo-core/src/main/java/com/ubiqube/etsi/mano/nfvo/notification/vnfind/VnfIndicatorValueChangeNotificationImpl.java:388]: Why in VNF indicator, we are talking about NS ?

**Review by Vignaudo — CHANGES_REQUESTED** [review]

File: nfvo-core/src/main/java/com/ubiqube/etsi/mano/nfvo/notification/vnfind/VnfIndicatorValueChangeNotificationImpl.java (hunk 2 above)
Comment (PrasadhNanjundan05) [reply nfvo-core/src/main/java/com/ubiqube/etsi/mano/nfvo/notification/vnfind/VnfIndicatorValueChangeNotificationImpl.java:388]: probably we will change the class name later,     it is one scheduler that doe both  vnf and ns auto scale.

Comment (Vignaudo) [reply nfvo-core/src/main/java/com/ubiqube/etsi/mano/nfvo/notification/vnfind/VnfIndicatorValueChangeNotificationImpl.java:388]: No it's ok, It's just I didn't read the package.

**Pull request closed without merging**